test/scripts/test_valhalla_build_config.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+import unittest
+
+try:
+    from test.scripts import helper
+except ModuleNotFoundError:
+    import helper
+
+valhalla_build_config = helper.import_path('valhalla_build_config')
```

Comment:
basically this is the same as saying `from valhalla.scripts import valhalla_build_config`